[
  {
    "type": "value_error",
    "loc": [
      "contributor"
    ],
    "msg": "Value error, At least one contributor must have role ContactPerson",
    "input": [
      {
        "name": "dkjljslkdfjlsd",
        "roleName": [],
        "schemaKey": "Organization",
        "contactPoint": [],
        "includeInCitation": false
      }
    ],
    "ctx": {
      "error": "At least one contributor must have role ContactPerson"
    },
    "url": "https://errors.pydantic.dev/2.10/v/value_error"
  }
]